src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -615,6 +615,9 @@ CharMap =
 #    mpegps   MPEG Program Stream                           video/mpeg                         Used in DVDs and VCDs
 #    mpegts   MPEG Transport Stream                         video/mpeg                         Used in sat TV and Blu-ray Discs
 #    mpegts   MPEG Transport Stream                         video/vnd.dlna.mpeg-tts            Alternative MIME-type for MPEG TS
+#    oga      Ogg container with only audio content         audio/ogg
+#    ogg      Ogg container                                 video/ogg                          Applies for Vorbis I files only, use with FLAC and Theora should use oga and ogv respectively
+#    ogv      Ogg container with video content              video/ogg
```

Comment:
Sorry, i thought it was included in [this PR](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1274), but in fact it's in the 2 audio PRs that are not ready to be pushed :flushed: 

That's said i could add it to the wma10 PR that i've already forgotten.